src/Transform/ONNX/ConstProp.cpp

```diff
@@ -404,6 +412,129 @@ Value ConstPropReduce(PatternRewriter &rewriter, Value replacingValue,
   }
 }
 
+//===----------------------------------------------------------------------===//
+// Code to perform constant propagation for matrix multiplication.
+//===----------------------------------------------------------------------===//
+
+// Takes the matrix shape and zero point for the LHS argument to MatMulInteger
+// and returns the zero point if it broadcasts to the matrix shape or else
+// returns the zero point reshaped so it broadcasts to the matrix shape.
+ElementsAttr reshapeMatMulIntegerLhsZero(
+    ArrayRef<int64_t> matrixShape, ElementsAttr zeroPoint) {
+  ShapedType zeroPointType = zeroPoint.getType();
+  auto zeroPointShape = zeroPointType.getShape();
```

Comment:
I think we try to not overuse `auto` as LLVM coding standard: https://llvm.org/docs/CodingStandards.html#use-auto-type-deduction-to-make-code-more-readable. In this case, I guest type is `ArrayRef<int64_t>`

Reply:
ok, changed in PR #2140